src/modelfree/shaping_wrappers.py

```diff
@@ -80,8 +91,13 @@ def step_wait(self):
 
             if done[env_num]:
                 for rew_type in REW_TYPES:
-                    self.ep_rew_dict[rew_type].append(sum(self.step_rew_dict[rew_type][env_num]))
+                    rew_type_total = sum(self.step_rew_dict[rew_type][env_num])
+                    self.ep_logs[rew_type].appendleft(rew_type_total)
                     self.step_rew_dict[rew_type][env_num] = []
+                # manually curate episode length because ConditionalAnnealers may want it
+                self.ep_logs['length'].appendleft(self.ep_len_dict[env_num])
```

Comment:
Is `ep_len_dict[env_num]` ever different to `len(self.step_rew_dict[rew_type][env_num])`? If so, why? If not, do we really need `ep_len_dict`?